src/Core/Framework/JWT/Constraints/HasValidRSAJWKSignature.php

```diff
@@ -41,7 +41,7 @@ public function assert(Token $token): void
             InMemory::plainText($pem)
         );
 
-        $configuration->setValidationConstraints(
+        $configuration->withValidationConstraints(
```

Comment:
this constraint was useless and never validated, @mstegmeyer was that intended?

Reply:
rolled the "fix" back as it broke a lot of unit tests to actually validate the constraint, will create a follow up ticket for the team

Reply:
will be double checked with https://shopware.atlassian.net/browse/NEXT-40456